### ISSUE TITLE
Trigger local script/command on rpi

### ISSUE THREAD

**Issue by cucubits** [alexa/avs-device-sdk#571]
Is there a way to use Alexa sample running on a raspberry pi to trigger a local script or command?

Obvious options with an actual echo device would be ha-bridge or fauxmo but it appears that both of these do not work with the sample app.

More specifically, I'm running the latest sample app sdk (1.6) on a raspberry pi3 and ideally I'd have it trigger a script when telling her "Alexa, turn on lights". This would be an easy task with above mentioned options but the sample app does not discover the new created virtual devices.

TLDR: How to trigger a simple command with the sample app running on the same raspberry pi?

**Comment by sanjayrd** [comment]
Hi @cucubits,

It sounds like you want to make Alexa do certain things when someone says a specific phrase like "turn on the lights".

You have two options:
1. Use Alexa Skills - this will send the audio data to the AVS cloud (allowing the cloud to perform the ASR and NLU processing), which in turn will trigger your Lambda script running in the cloud. This is the easier route to go, although it sounds like you specifically want to trigger a local script.

2. Intercept audio containing specific phrases locally on-device with your own local ASR, and then route it to your own local script, rather than sending it to AVS. This is a bit more complicated and would require (probably significant) changes to the SDK/Sample App code.

Hope that helps,
Sanjay

**Comment by dhpp** [comment]
Closing.   Please re-open if you require further guidance on this.
